index.md

```diff
@@ -101,6 +101,10 @@ Selskapet har pensjonssparing hos Gjensidige for alle ansatte. Vi innbetaler 5%
 
 Det er åpnet opp for at de ansatte kan samle sine (tidligere) pensjonsavtaler hos Gjensidige. Hver ansatt må kontakte Gjensidige om dette.
 
+### Foreldrepermisjon
+
+Det å bli foreder er en fantastisk opplevelse og vi unner alle å ta en velfortjent foreldrepermisjon. Vi betaler full lønn i foreldrepermisjon og tiden du er i permisjon vil ikke gå utover din ansiennitet i selskapet.
```

Comment:
👍 Bare en liten trykkleif på "foreder".

Reply:
Endret betydningen ganske mye, ja :) Skal fikse

Reply:
For personer som ikke har satt seg inn i foreldrepenger-reglene, vil det ikke være litt gitt at vi betaler full lønn i permisjonen? Burde vi presisert at "321 dekker eventuell differanse mellom det du får fra NAV og full lønn"?

Reply:
Jeg liker at teksten er såpass enkel og grei :-)

Reply:
Enig i denne formuleringen, uten trykkleif da :)